File: CoffeeShopManager.API/Models/Member.cs
using System;

namespace CoffeeShopManager.API.Models
{
    public class Member
    {
        public int Id { get; set; }

        public string Name { get; set; }

        public string Gender { get; set; }

        public string Phone { get; set; }

        public DateTime DateOfBirth { get; set; }

        public string Address { get; set; }

        public DateTime CreatedDate { get; set; }

        public string Photo { get; set; }
    }
}

File: CoffeeShopManager-SPA/src/app/_models/Member.ts
import { Photo } from './Photo';

export interface Member {
    id: number;
    name: string;
    age: number;
    photo: string;
    gender: string;
    phone: string;
    address: string;
    isDelete: boolean;
    dateOfBirth: Date;
    createdDate: Date;
    photos?: Photo[];
    point: number;
}
